content/doc/book/pipeline/getting-started.adoc

```diff
@@ -22,106 +23,198 @@ This section introduces some of the key concepts to Jenkins Pipeline and
+  Pipeline directly in Jenkins through the classic UI.
+* <<#defining-a-pipeline-in-scm,In SCM>> - you can write a `Jenkinsfile`
+  manually, which you can commit to your project's source control repository.
 
 The syntax for defining a Pipeline with either approach is the same, but while
-Jenkins supports entering Pipeline directly into the web UI, it's
+Jenkins supports entering Pipeline directly into the classic UI, it is
 generally considered best practice to define the Pipeline in a `Jenkinsfile`
 which Jenkins will then load directly from source control.
-footnoteref:[scm, https://en.wikipedia.org/wiki/Source_control_management]
-
-
-=== Defining a Pipeline in the Web UI
-
-To create a basic Pipeline in the Jenkins web UI, follow these steps:
-
-* Click *New Item* on Jenkins home page.
-
-image::pipeline/new-item-selection.png["Click *New Item* on the Jenkins home page", role=center]
-
-* Enter a name for your Pipeline, select *Pipeline* and click *OK*.
-
-[CAUTION]
-====
-Jenkins uses the name of the Pipeline to create directories on disk. Pipeline
-names which include spaces may uncover bugs in scripts which do not expect
-paths to contain spaces.
-====
-
-image::pipeline/new-item-creation.png["Enter a name, select *Pipeline*, and click *OK*", role=center]
-
-
-
-* In the *Script* text area, enter a Pipeline and click *Save*.
-
-image::pipeline/hello-world-script.png["In the *Script* text area, enter a Pipeline and click Save", role=center]
-
-* Click *Build Now* to run the Pipeline.
-
-image::pipeline/build-now.png["Click *Build Now* to run the Pipeline", role=center]
-
-
-* Click *#1* under "Build History" and then click *Console Output* to see the
-  full output from the Pipeline.
-
-image::pipeline/hello-world-console-output.png["Click *Console Output* for the Pipeline", role=center]
-
-The example above shows a successful run of a basic Pipeline created in the Jenkins
-web UI, using two steps.
-
+footnoteref:[scm,link:https://en.wikipedia.org/wiki/Source_control_management[Source control management]]
+
+
+=== Through Blue Ocean
+
+If you are new to Jenkins Pipeline, the Blue Ocean UI helps you
+<<../blueocean/creating-pipelines#,set up your Pipeline project>>, and
+automatically creates and writes your Pipeline (i.e. the `Jenkinsfile`) for you
+through the graphical Pipeline editor.
+
+As part of setting up your Pipeline project in Blue Ocean, Jenkins configures a
+secure and appropriately authenticated connection to your project's source
+control repository. Therefore, any changes you make to the `Jenkinsfile` via
+Blue Ocean's Pipeline editor are automatically saved and committed to source
+control.
+
+Read more about Blue Ocean in the <<../blueocean#,Blue Ocean>> chapter and
+<<../blueocean/getting-started#,Getting started with Blue Ocean>> page.
+
+
+=== Through the classic UI
+
+`Jenkinsfile`&zwj;s created using the classic UI are stored by Jenkins itself
```

Comment:
Could use the phrase:

> A Jenkinsfile created ... is ...

rather than 

> Jenkinsfile's created ... are ...

Since the most common case is a single Jenkinsfile per branch of a repository.

Reply:
Good point - I think the singular construction works better than the pluralised one - especially  considering the fact that the procedure explaining this text involves creating only a single `Jenkinsfile`.

Having said that, AFAICT, this part of the documentation only refers to a `Jenkinsfile` which is not housed in SCM but only somewhere in the Jenkins home directory.

The **In SCM** section of this page documents the `Jenkinsfile`-housed-in-SCM procedure.